switchapi/es2k/switch_device.c

```diff
@@ -26,6 +26,7 @@
 #include "switchapi/switch_rmac_int.h"
 #include "switchapi/switch_table.h"
 #include "switchapi/switch_vrf.h"
+#include "switchapi/switch_lag.h"
```

Comment:
General comment for all header file inclusions, keep it alphabetial order. 
switch_lag.h should have been after switch_internal.h.

Reply:
Addressed.